integration/macos/publish.sh

```diff
@@ -14,7 +14,7 @@ set -ex
 
 cat <<EOF >"${HOME}/.ssh/config"
 Host upload-packages.edgedb.com
-    Port 2224
+    Port 2222
```

Comment:
Curious: why the port switcheroo?

Reply:
I just set this up in the wrong order in the LB on AWS and now switching the ports up is a hassle, we'd need to recreate the LB and the service.